### DISCUSSION TITLE
Filter for `relativePath` of references on *Connections

### DISCUSSION THREAD

**Discussion by Fox32** [tinacms/tinacms#3754] [Ideas]
I'm playing around with references and I think it's quite a strong feature. But I feel quite limited right now and have to find smart/clever tricks to do stuff that in my opinion should be built-in. I will walk you through my example:

I have two collections, a `post` and an `author` collection. The post collection contains a reference field called `postAuthor` that references an entry from the `author` collection. The `author` collection has a `displayName`.

Here are some typical use cases I encountered:

**I want to show the authors display name within my post**: This is a common use case and Tina already supports it well by allowing to embed the author entity inside the post. All data is at hand, well done!

**I want to display all posts of an author on his summary page**: This is more difficult, the best option I have is to use the `postConnection` and filter by author. But there my problem begins: There are filters for the `postAuthor` field available, however they can only filter on the entity data, like the display name. I could be smart and just filter by display name, but that doesn't have to be a unique field, or I can create an additional unique id field that I have to populate. It would be better to filter by `relativePath` instead:

```graphql
postConnection(filter: {author: {relativePath: "my-filename.md"}}) {
  …
}
```

So my suggestion is to add a `StringFilter` called `relativePath` to `PostPostAuthorFilter`.

**Comment by jeffsee55** [comment]
One of the things on our roadmap is to just be able to lookup references in the other direction. So you'd just be able to get all of the author's posts like this:

```graphql
query {
  author(relativePath: "napolean.md") {
    name
    references {
      post {
        edges {
          node {
             title
          }
        }
      }
    }
  }
}
```

We're not there yet, but this seems like it'd be consistent with what you need

**Reply by Fox32** [reply]
Would be great too, it would feel very much like a graphql schema should be. 

My concern there is that a post that might have different references to authors might make the things very complex. Imaging a post that has both a creator and a reviewer reference into the authors collection.

**Reply by jeffsee55** [reply]
Yeah agreed, we've discussed a few possibilities about how to handle that but it will likely be customizable
